packages/hash/api/src/graphql/typeDefs/ontology/entity-type.typedef.ts

```diff
@@ -0,0 +1,56 @@
+import { gql } from "apollo-server-express";
+
+export const entityTypeTypedef = gql`
+  scalar EntityType
+
+  type PersistedEntityType {
+    entityTypeVersionedUri: String!
+    createdBy: ID!
+    schema: EntityType!
+    # TODO: we might need something like
+    # "referencedPropertyTypes: [PersistedPropertyType!]"
```

Comment:
Perhaps this could be something that @Alfred-Mountfield's WASM library could help obtain on the FE?

Reply:
Yes, somewhat. This resolver would also fetch out the instances of the referenced properties, not just their URIs. That library would definitely help if we want the fetch logic to happen in this API rather than the Graph

Reply:
That's a good point, let's keep the comment here then 👍